config/config.go

```diff
@@ -88,7 +107,10 @@ func (c Config) Save(path string) (err error) {
 // 3.  The value of $XDG_CONFIG_PATH/func if the environment variable exists.
 // The path is created if it does not already exist.
 func Path() (path string) {
-	path = DefaultConfigPath
+	// default path is a relative path used in the unlikely event that
+	// the user has no home directory (no ~), there is no
+	// XDG_CONFIG_HOME set
+	path = filepath.Join(".config", "func")
```

Comment:
This is a bit of an arbitrary choice.  We might want to consider having the deafult path be simply "." in this admitedly quite edge edge case